src/graph/CreateSpaceExecutor.cpp

```diff
@@ -23,13 +23,17 @@ Status CreateSpaceExecutor::prepare() {
             case SpaceOptItem::PARTITION_NUM:
                 partNum_ = item->get_partition_num();
                 if (partNum_ <= 0) {
-                    return Status::Error("Partition_num value should be greater than zero");
+                    return Status::Error("partition_num value should be greater than zero");
+                } else if (partNum_ >=100000) {
```

Comment:
If you really want to limit the max numbers,  use one gflag to control it.  And the logic should be implemented in metad.

Reply:
ok i understand